scripts/Commands/LintStagedCommand.php

```
@@ -0,0 +1,188 @@
+            $output->writeln('Fixes applied. Adding fixed files to the staging area...');
+
+            if (!$this->addFilesToStaging($phpFiles, $output)) {
+                $output->writeln('<error>Failed to add fixed files to the staging area.</error>');
+
+                return Command::FAILURE;
+            }
+
+            $output->writeln('Fixed files added to the staging area.');
+        } else {
+            $output->writeln('No fixes needed.');
+        }
```

PHPMD (2.15.0)

[error] The method runLintStaged uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)